### PR TITLE
Update intro.rst to Turkish

### PR DESCRIPTION
**Pull request by ugurozturk**

### CODE REVIEW

File: tr/tutorials-and-examples/bookmarks/intro.rst [hunk 1]
```diff
@@ -1,9 +1,370 @@
...
+
+Kullanıcılarınızı oluşturduktan sonra (**http://localhost:8765/bookmarks/users** sayfasında ki), siz muhtemelen
+şifrelerin düz metin olarak saklandığını görüyorsunuzdur. Bu gözle görünür derecede oldukça güvenlik açısından kötüdür. Yani bunu düzeltmemiz gerekiyor. 
+
+Aynı zamanda CakePHP deki model yapımız hakkında konuşma zamanımız geldi. CakePHP de ayrı metotları ayırarak nesne üstünde birleştiriyoruz ve bir neste içerisinde farklı sınıflar oluyor.
+Yöntemleri ``Table`` sınıflarını içeriye atarak topluyoruz. Bu özellikleri de tek başına ``Entity`` sınıfına aktardık.
+
+Örnek olarak, şifre karıştırma işlemi tamamen başlı başına bir  bireysel kayıttır, yani entity nesnesin deki işlevi uygulayacağız. Çünkü biz şifrelerimizi her kayıt olduğu an karıştırmak istiyoruz.
+Biz mutator/setter yöntemini uygulayacağız.Her hangi bir zaman entities deki bir özellik çalıştığı zaman CakePHP setter metotunu çağıracaktır. Şifrelerimiz için setter'i ekleyelim.
+**src/Model/Entity/User.php** de şunları izleyin.
```
Comment (markstory) [inline tr/tutorials-and-examples/bookmarks/intro.rst]: This line needs to end `::` to make the following code block format correctly.

**Comment by cake17** [comment]
@midorikocak It would be nice if you could review this, I can't help on this 😄

**Comment by cake17** [comment]
and thanks @ugurozturk for your contribution 👍

**Comment by andtxr** [comment]
The line length is over 80 characters in many points. We can handle it if the translation is ok. @ugurozturk  thanks so much for the help, perhaps you would like to take a look in the [documentation contribution guidelines](http://book.cakephp.org/3.0/en/contributing/documentation.html).

**Comment by ugurozturk** [comment]
@andtxr Oh thanks. I just tried to translate it from english doc to turkish.

**Pull request merged**